include/intx/int128.hpp

```diff
@@ -103,6 +107,16 @@ struct uint<128>
 using uint128 = uint<128>;
 
 
+inline constexpr bool is_constant_evaluated() noexcept
+{
+#if __has_builtin(__builtin_is_constant_evaluated)
```

Comment:
Unfortunately, this will eliminate MSVC although it supports `__builtin_is_constant_evaluated` but not `__has_builtin`. I think here you should add old-fashioned `_MSC_VER` check also.